transformations/syntactically_diverse_paraphrase/transformation.py

```diff
@@ -0,0 +1,55 @@
+import benepar
+import spacy
+import nltk
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+from transformations.syntactically_diverse_paraphrase.sowreap.parse_utils import (
+    Sentence,
+)
+from transformations.syntactically_diverse_paraphrase.sowreap.reap_utils import (
+    reapModel,
+)
+from transformations.syntactically_diverse_paraphrase.sowreap.sow_utils import (
+    sowModel,
+)
+
+
+class ParaphraseSowReap(SentenceOperation):
```

Comment:
Because of the number of very large models you have to load, I think you have to mark this class with the attribute `heavy = True`. This attribute will make it so the transformation is not tested as part of the CI run.

As it is the unit tests are timing out due to the long load times for the models.

Reply:
Done.